nav2_behavior_tree/plugins/action/navigate_to_pose_action.cpp

```diff
@@ -39,28 +39,15 @@ class NavigateToPoseAction : public BtActionNode<nav2_msgs::action::NavigateToPo
 
   void on_tick() override
   {
-    // Use the position and orientation fields from the XML attributes to initialize the goal
-    geometry_msgs::msg::Point position;
-    geometry_msgs::msg::Quaternion orientation;
-
-    if (!getInput("position", position) || !getInput("orientation", orientation)) {
```

Comment:
Why did you remove error handling?

Reply:
Thanks for reviewing. I was careless.  
I've restored it.